emscripten-releases-tags.txt

```diff
@@ -1,5 +1,7 @@
 {
```

Comment:
Btw, not that it matters, but why does this file use `.txt` extension rather than JSON?

Reply:
You are right.. we should rename them